src/zsys.c

```diff
@@ -912,7 +912,9 @@ zsys_udp_recv (SOCKET udpsock, char *peername, int peerlen)
 {
     char buffer [UDP_FRAME_MAX];
     in6addr_t address6;
+#if (!defined (__WINDOWS__))
     inaddr_t *address = (inaddr_t *) &address6;
+#endif
```

Comment:
Could you please put this into a second commit. You don't need to open another PR. For commit message we do use the Problem/Solution statement, e.g.:

Problem: address variable in zsys is unused in windows build
Solution: only define variable when build os is not windows